_observing-your-data/alerting/composite-monitors.md

```diff
@@ -63,7 +63,7 @@ In this simple example, the first monitor could be a per document monitor config
 
 ## Managing composite monitors with the API
 
-You can manage composite monitors using the REST API or OpenSearch Dashboards. This section covers API functionality for composite monitors.
+You can manage composite monitors using the OpenSearch REST API or [OpenSearch Dashboards](#creating-composite-monitors-in-opensearch-dashboards). This section covers API functionality for composite monitors.
```

Comment:
Minor suggestion: Because "covers" can have more than one meaning, would "describes" be better?